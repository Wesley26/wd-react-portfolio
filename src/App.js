import React from 'react';
import {
  BrowserRouter as Router,
  Switch,
  Route
} from 'react-router-dom';
import './index.css';
import './tailwind.css';
import Footer from './components/Footer';
import Header from './components/Header';
import About from './webPages/About.js';
import Home from './webPages/Home.js'

function App() {
  return (
    <div className="bg-body-gray absolute bottom-0 top-0">
      <Router>

      <main className="html">
        <div className="text-center">
          <Header />

          <Switch>
            <Route exact path="/">
              <Home />
            </Route>
            <Route exact path="/about">
              <About />   
            </Route>
          </Switch>

        </div>

      </main>

      <Footer />
      </Router>

    </div>
  );
}

export default App;
