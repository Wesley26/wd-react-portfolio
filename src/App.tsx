import { FC } from 'react';
import {
  BrowserRouter as Router,
  Switch,
  Route
} from 'react-router-dom';
import './index.css';
import Footer from './components/Footer';
import Header from './components/Header';

import { About,
          ContactMe,
          Home,
          Portfolio,
          Resume,
          TodoList,
       } from './webPages/index';

const App:FC = () => {
  return (
    <div className="bg-body-gray absolute bottom-0 top-0 left-0 right-0">
      <Router>

      <main className="bg-body-gray html border-solid border-gray-300 border-t-4 border-l-4 border-r-4 pb-24">
        <div className="text-center">
          <Header />

          <Switch>
            <Route exact path="/">
              <Home />
            </Route>
            <Route exact path="/about">
              <About />   
            </Route>
            <Route exact path="/portfolio">
              <Portfolio />   
            </Route>
            <Route exact path="/resume">
              <Resume />
            </Route>
            <Route exact path="/contactMe">
              <ContactMe />
            </Route>
            <Route exact path="/todoList">
              <TodoList />
            </Route>
          </Switch>

        </div>

      </main>

      <Footer />
      </Router>

    </div>
  );
}

export default App;
