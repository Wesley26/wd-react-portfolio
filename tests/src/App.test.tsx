import { render } from '@testing-library/react';
import App from '../../src/App';
import '../../src/index.css';

it(`render App.tsx test component`, () => {

    const view = (render(<App />));

    expect(view).toMatchSnapshot();

});