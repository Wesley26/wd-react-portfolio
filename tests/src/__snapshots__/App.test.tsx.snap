// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App.tsx test component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="bg-body_gray absolute bottom-0 top-0 left-0 right-0"
      >
        <main
          class="bg-body_gray html border-solid border-gray-300 border-t-4 border-l-4 border-r-4 pb-24"
        >
          <div
            class="text-center"
          >
            <header
              class="bg-header_blue mb-3 pl-3 pr-3 pt-1 pb-6 flex justify-between shadow-lg sm:pb-32"
            >
              <h1>
                <span
                  class="font-title pt-3 pl-3 font-bold float-left"
                >
                  <div
                    class="grid grid-cols-1 items-center md:text-2xl"
                  >
                    <img
                      alt="Image has failed to load."
                      class="h-12 w-24"
                      src="[object Object]"
                    />
                    Wesley Dzitzer
                  </div>
                </span>
                <nav>
                  <span
                    class="font-body mr-3 pt-3 pb-3 pr-3 pl-12 absolute right-0 float-right text-l"
                  >
                    <div
                      class="flex flex-row-reverse"
                    >
                      <div
                        class="h-22 text-green-600 text-xl pt-6 pb-3 px-6 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                        style="opacity: 0; transform: translateX(-100px) translateY(0px) translateZ(0);"
                      >
                        <p
                          class="pb-3"
                        >
                          Apps
                        </p>
                      </div>
                    </div>
                    <div
                      class="invisible hidden h-auto sm:pt-1 sm:visible sm:flex sm:flex-1 sm:justify-center sm:items-center"
                      style="opacity: 0; transform: translateX(-100px) translateY(0px) translateZ(0);"
                    >
                      <div
                        class="flex h-20 md:w-34long sm:w-32long"
                      >
                        <div
                          class="flex flex-row font-title pb-12 justify-evenly"
                        >
                          <div
                            class="pt-3"
                          >
                            <a
                              class="bg-important_blue h-5.9long text-green-600 text-xl mt-0.10long pt-8 pb-3.5long px-4 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                              href="/"
                            >
                              Home
                            </a>
                          </div>
                          <div
                            class="px-1"
                          />
                          <div
                            class="pt-3"
                          >
                            <a
                              class="bg-transparent h-22 text-green-600 text-xl pt-6 pb-3 px-2 mb-6 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                              href="/about"
                            >
                              About Me
                            </a>
                          </div>
                          <div
                            class="px-1"
                          />
                          <div
                            class="pt-3"
                          >
                            <a
                              class="bg-transparent h-22 text-green-600 text-xl pt-6 pb-3 px-2 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                              href="/portfolio"
                            >
                              Portfolio Gallery
                            </a>
                          </div>
                          <div
                            class="px-1"
                          />
                          <div
                            class="pt-3"
                          >
                            <a
                              class="bg-transparent h-5.9long text-green-600 text-xl mt-0.10long pt-8 pb-3.5long px-2 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                              href="/resume"
                            >
                              Resume
                            </a>
                          </div>
                          <div
                            class="px-1"
                          />
                          <div
                            class="pt-3"
                          >
                            <a
                              class="bg-transparent h-22 text-green-600 text-xl pt-6 pb-3 px-2 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                              href="/contactMe"
                            >
                              Contact Me
                            </a>
                          </div>
                        </div>
                      </div>
                    </div>
                  </span>
                </nav>
              </h1>
            </header>
            <div
              class="font-body bg-body_lightGray m-6 shadow-xl"
            >
               
              <div
                class="p-6"
                style="opacity: 0; transform: translateX(0px) translateY(-20px) translateZ(0);"
              >
                <div
                  class="pb-10"
                >
                  <div
                    class="flex justify-center items-center"
                  >
                    <img
                      alt="Image has failed to load."
                      class="focus:outline-none"
                      src="[object Object]"
                    />
                  </div>
                  <h3
                    class="font-title text-xl text-center font-bold"
                  >
                    Web and Mobile Design: Perfectly Balanced To Meet Your Future Business Image
                  </h3>
                  <br />
                  <p
                    class="text-lg"
                  >
                    Are you searching for a new, high-tech, scalable web and/or mobile design to welcome your customers?
                            Do you have software engineers on your current team who specialize behind the scenes, but do 
                            not work as often on what the end-user views? I can help you! Feel free to browse my portfolio 
                            gallery for all the examples of web and mobile design I can provide to your software engineer 
                            team! Use the menu button on the top right of your screen to begin.
                  </p>
                  <br />
                  <div>
                    <figure>
                      <img
                        alt="Image has failed to load."
                        class="lg:hidden focus:outline-none shadow-lg"
                        src="[object Object]"
                      />
                      <img
                        alt="Image has failed to load."
                        class="invisible h-1 w-1 lg:visible lg:h-full lg:w-full focus:outline-none shadow-lg"
                        src="[object Object]"
                      />
                      <figcaption
                        class="pt-2 text-left"
                      >
                        The photo above was taken in person outside the South-East doorway of the Philadelphia Art Museum overlooking 
                                the famed Rocky Steps in Philadelphia, P.A.
                      </figcaption>
                    </figure>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </main>
        <div>
          <footer
            class="font-body bg-footer_blue border-solid border-gray-300 border-b-4 border-l-4 border-r-4 text-center text-s p-3 fixed bottom-0 w-full shadow-lg"
          >
            Â© 
            Copyright 2020 -  2022
            <br />
            The current date of server is:  3/1/2022
            <br />
            The current time of server is: 5:28:32 AM
          </footer>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="bg-body_gray absolute bottom-0 top-0 left-0 right-0"
    >
      <main
        class="bg-body_gray html border-solid border-gray-300 border-t-4 border-l-4 border-r-4 pb-24"
      >
        <div
          class="text-center"
        >
          <header
            class="bg-header_blue mb-3 pl-3 pr-3 pt-1 pb-6 flex justify-between shadow-lg sm:pb-32"
          >
            <h1>
              <span
                class="font-title pt-3 pl-3 font-bold float-left"
              >
                <div
                  class="grid grid-cols-1 items-center md:text-2xl"
                >
                  <img
                    alt="Image has failed to load."
                    class="h-12 w-24"
                    src="[object Object]"
                  />
                  Wesley Dzitzer
                </div>
              </span>
              <nav>
                <span
                  class="font-body mr-3 pt-3 pb-3 pr-3 pl-12 absolute right-0 float-right text-l"
                >
                  <div
                    class="flex flex-row-reverse"
                  >
                    <div
                      class="h-22 text-green-600 text-xl pt-6 pb-3 px-6 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                      style="opacity: 0; transform: translateX(-100px) translateY(0px) translateZ(0);"
                    >
                      <p
                        class="pb-3"
                      >
                        Apps
                      </p>
                    </div>
                  </div>
                  <div
                    class="invisible hidden h-auto sm:pt-1 sm:visible sm:flex sm:flex-1 sm:justify-center sm:items-center"
                    style="opacity: 0; transform: translateX(-100px) translateY(0px) translateZ(0);"
                  >
                    <div
                      class="flex h-20 md:w-34long sm:w-32long"
                    >
                      <div
                        class="flex flex-row font-title pb-12 justify-evenly"
                      >
                        <div
                          class="pt-3"
                        >
                          <a
                            class="bg-important_blue h-5.9long text-green-600 text-xl mt-0.10long pt-8 pb-3.5long px-4 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                            href="/"
                          >
                            Home
                          </a>
                        </div>
                        <div
                          class="px-1"
                        />
                        <div
                          class="pt-3"
                        >
                          <a
                            class="bg-transparent h-22 text-green-600 text-xl pt-6 pb-3 px-2 mb-6 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                            href="/about"
                          >
                            About Me
                          </a>
                        </div>
                        <div
                          class="px-1"
                        />
                        <div
                          class="pt-3"
                        >
                          <a
                            class="bg-transparent h-22 text-green-600 text-xl pt-6 pb-3 px-2 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                            href="/portfolio"
                          >
                            Portfolio Gallery
                          </a>
                        </div>
                        <div
                          class="px-1"
                        />
                        <div
                          class="pt-3"
                        >
                          <a
                            class="bg-transparent h-5.9long text-green-600 text-xl mt-0.10long pt-8 pb-3.5long px-2 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                            href="/resume"
                          >
                            Resume
                          </a>
                        </div>
                        <div
                          class="px-1"
                        />
                        <div
                          class="pt-3"
                        >
                          <a
                            class="bg-transparent h-22 text-green-600 text-xl pt-6 pb-3 px-2 flex border-black border-solid border-2 rounded-md hover:bg-gray-300 active:bg-gray-400 shadow-lg"
                            href="/contactMe"
                          >
                            Contact Me
                          </a>
                        </div>
                      </div>
                    </div>
                  </div>
                </span>
              </nav>
            </h1>
          </header>
          <div
            class="font-body bg-body_lightGray m-6 shadow-xl"
          >
             
            <div
              class="p-6"
              style="opacity: 0; transform: translateX(0px) translateY(-20px) translateZ(0);"
            >
              <div
                class="pb-10"
              >
                <div
                  class="flex justify-center items-center"
                >
                  <img
                    alt="Image has failed to load."
                    class="focus:outline-none"
                    src="[object Object]"
                  />
                </div>
                <h3
                  class="font-title text-xl text-center font-bold"
                >
                  Web and Mobile Design: Perfectly Balanced To Meet Your Future Business Image
                </h3>
                <br />
                <p
                  class="text-lg"
                >
                  Are you searching for a new, high-tech, scalable web and/or mobile design to welcome your customers?
                            Do you have software engineers on your current team who specialize behind the scenes, but do 
                            not work as often on what the end-user views? I can help you! Feel free to browse my portfolio 
                            gallery for all the examples of web and mobile design I can provide to your software engineer 
                            team! Use the menu button on the top right of your screen to begin.
                </p>
                <br />
                <div>
                  <figure>
                    <img
                      alt="Image has failed to load."
                      class="lg:hidden focus:outline-none shadow-lg"
                      src="[object Object]"
                    />
                    <img
                      alt="Image has failed to load."
                      class="invisible h-1 w-1 lg:visible lg:h-full lg:w-full focus:outline-none shadow-lg"
                      src="[object Object]"
                    />
                    <figcaption
                      class="pt-2 text-left"
                    >
                      The photo above was taken in person outside the South-East doorway of the Philadelphia Art Museum overlooking 
                                the famed Rocky Steps in Philadelphia, P.A.
                    </figcaption>
                  </figure>
                </div>
              </div>
            </div>
          </div>
        </div>
      </main>
      <div>
        <footer
          class="font-body bg-footer_blue border-solid border-gray-300 border-b-4 border-l-4 border-r-4 text-center text-s p-3 fixed bottom-0 w-full shadow-lg"
        >
          Â© 
          Copyright 2020 -  2022
          <br />
          The current date of server is:  3/1/2022
          <br />
          The current time of server is: 5:28:32 AM
        </footer>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
